index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute page</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div id="main">
+        <h1 id="title" class="name">Andre Acosta</h1><br>
+        <p>The Best Coding Partner</p><br>
+        <div id="img-div">
+            <img id="img" class="responsive" src="https://www.oficinadanet.com.br/imagens/post/25092/cc01.jpg" alt="Hero Image">
+            <p id="img-caption">The TimeLine of Andre's Life</p>
+        </div>
+    <div id="tribute-info">
+        <h1>Timeline</h1>
+        <ul>
+            <li><strong>2003</strong> - Born in Belize</li>
+            <li><strong>2010</strong> - Learned how to code</li>
+            <li><strong>2015</strong> - Phd at Computer Sciences</li>
+            <li><strong>2017</strong> - Reached the top 10 billionaires</li>
+            <li><strong>2020</strong> - Colonized mars before Elon Musk</li>
+            <li><strong>2021</strong> - Applied to Microverse</li>
+            <li><strong>2022</strong> - Started work at his own startup</li>
+
+        </ul>
+        <a id="tribute-link" href="https://www.wikipedia.org/
+        " target="_blank">Learn More</a>
+    </div>
+    </div>
+
+</body>
+</html>
```

Comment:
- [ ] [REQUIRED] I think it would be better if you phrased the comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"I think it would be better to use semantic tags"*
* This reads like an order: *"Use semantic tags"*😊😊